src/tools/autocompletion/spec

```diff
@@ -0,0 +1,140 @@
+[]
+root=kdb
+
+[kdb/backup]
+opt/long=backup
```

Comment:
These are commands and not long options. Better start with one tool and its short/long options.